### ISSUE TITLE
ValueError: Invalid model savp

### ISSUE THREAD

**Issue by chetankm1992** [alexlee-gk/video_prediction#5]
Hi, I am trying to run your code by following your instructions but there are some issues.
1) First one was that "no module named datasets and models" so I added a code snippet wherever it was required, that is:
import sys, os
sys.path.append('/home/chetan/Desktop/video_prediction/video_prediction')
and it is working now.
2) Then, I am getting another error that
Traceback (most recent call last):
  File "scripts/generate.py", line 184, in <module>
    main()
  File "scripts/generate.py", line 114, in main
    VideoPredictionModel = models.get_model_class(args.model)
  File "/home/chetan/Desktop/video_prediction/video_prediction/models/__init__.py", line 33, in get_model_class
    raise ValueError('Invalid model %s' % model)
ValueError: Invalid model savp

I tried to solve this issue but I am unable to do so. Can you please suggest me what should I do. 
Thank you.

**Comment by chetankm1992** [comment]
I commented below if condition in video_prediction/video_prediction/models/_init_.py file and the code is working fine. I don't know is it a good solution or not but for now things are working.
#if model_class is None or not issubclass(model_class, BaseVideoPredictionModel):
    #    raise ValueError('Invalid model %s' % model)

**Comment by alexlee-gk** [comment]
1. The python scripts should be run from the root directory or the root `video_prediction` directory should be added to the `PYTHONPATH`.
2. I don't see why it would throw that error since `'savp'` is a valid model. Is there any reason that file is named `init.py` instead of `__init__.py`?

**Comment by chetankm1992** [comment]
Thank you for your reply.
1. I was running the given script from root folder i.e, video_prediction but it was showing "no module named video_prediction" so I just added using sys.path.append, wherever it was required and now this error is resolved. By the way I tried adding directory using PYTHONPATH as you mentioned but still I am getting error, so I will stay with my earlier solution.
2. File name is `__init__.py`, but due to github auto text updating it was turned to init.py. I am getting "savp" error, I don't know why but as I mentioned when I commented if condition, code seems to be working and generating results but I am confused what is the prediction part and the ground truth part. So, I am just trying to figure it out.
Any suggestions for savp error and results understanding will be helpful.
